bucket/nim.json

```diff
@@ -5,20 +5,22 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://nim-lang.org/download/nim-0.20.2_x64.zip",
+            "url": "https://nim-lang.org/download/nim-0.20.2_x64.zip#/dl.7z",
             "hash": "ccd258ae90d2af2eb1042830f1204920e8d44b6947aedb56458110cf9a714207"
         },
         "32bit": {
-            "url": "https://nim-lang.org/download/nim-0.20.2_x32.zip",
+            "url": "https://nim-lang.org/download/nim-0.20.2_x32.zip#/dl.7z",
```

Comment:
Please revert this to `.zip`. This change https://github.com/lukesampson/scoop/pull/3460 will provide this functionality.

Reply:
Yeah, I know...but how to install nim before https://github.com/lukesampson/scoop/pull/3460 in master?

Reply:
The current manifest works using zip. 😕 

![image](https://user-images.githubusercontent.com/432127/64113531-da1c9d00-cd8a-11e9-9b3f-7f94e0bd221a.png)

Reply:
Stop! I'm using the develop branch. :grin:

Reply:
Still works with master branch:
![image](https://user-images.githubusercontent.com/432127/64113706-439cab80-cd8b-11e9-95de-45f16e2cdce7.png)

Reply:
Strange... It does not work for me. I got the same error as https://github.com/lukesampson/scoop/issues/3621 .

![2019-09-02 20_35_13-Windows PowerShell](https://user-images.githubusercontent.com/36977733/64115071-3a591200-cd7e-11e9-9234-262358b8baaa.png)

Reply:
What is the output of `$PSVersionTable`?

Reply:
I get the same error and an empty directory. 

```
D:\..\cache ❯❯❯ $psversiontable

Name                           Value
----                           -----
PSVersion                      5.1.18362.145
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18362.145
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

Reply:
@r15ch13 When I run `Microsoft.PowerShell.Archive\Expand-Archive -Path ".\nim#0.20.2#https_nim-lang.org_download_nim-0.20.2_x64.zip" -DestinationPath nim -Force` from the scoops cache directory, I got the same error as https://github.com/lukesampson/scoop/issues/3621

Reply:
@linsui What terminal do you use? It looks cool. Haha

Reply:
https://github.com/akkuman It's powershell with colortool and oh-my-posh :joy: .
@r15ch13 It works well in pscore 6.2.2 .

Reply:
hm, can't reproduce it in any way.

![image](https://user-images.githubusercontent.com/432127/64186695-8fc11c00-ce6f-11e9-8d46-a38730fe7d4f.png)

Reply:
Maybe Chinese windows 10

Reply:
@Rvn0xsy Maybe not, but related. I enable UTF8 and it works well now.

![](https://pic1.zhimg.com/80/v2-df928fbec72f4ec433cc0422f70b4944_hd.jpg)